docs/tutorials/getting_started/tournament_results.rst

```diff
@@ -0,0 +1,258 @@
+- Score differences: the score difference between each player.
+- Payoff difference means: the mean score differences.
+- Cooperation counts: the number of times each player cooperated.
+- Normalised cooperation: cooperation count per turn.
+- Cooperation rating: cooperation rating of each player
+- Vengeful cooperation: a morality metric from the literature (see
+  :ref:`morality-metrics`).
+- Good partner matrix: a morality metric from the literature.
+- Good partner rating: a morality metric from the literature.
+- Eigenmoses rating: a morality metric from the literature.
+- Eigenjesus rating: a morality metric from the literature.
+
+As shown in :ref:`creating_tournaments` let us create a tournament::
+
+    >>> import axelrod as axl
+    >>> players = [axl.Cooperator(), axl.Defector(),
+    ...            axl.TitForTat(), axl.Grudger()]
+    >>> tournament = axl.Tournament(players, turns=10, repetitions=3)
+    >>> results = tournament.play()
+
+Wins
+----
+
+This gives the number of wins obtained by each player::
+
+    >>> results.wins
+    [[0, 0, 0], [3, 3, 3], [0, 0, 0], [0, 0, 0]]
+
+
+The :code:`Defector` is the only player to win any matches (all other matches
+are ties).
+
+Match lengths
+-------------
+
+This gives the length of the matches played by each player::
+
+    >>> import pprint  # Nicer formatting of output
+    >>> pprint.pprint(results.match_lengths)
+    [[[10, 10, 10, 10], [10, 10, 10, 10], [10, 10, 10, 10], [10, 10, 10, 10]],
+     [[10, 10, 10, 10], [10, 10, 10, 10], [10, 10, 10, 10], [10, 10, 10, 10]],
+     [[10, 10, 10, 10], [10, 10, 10, 10], [10, 10, 10, 10], [10, 10, 10, 10]]]
+
+Every player plays 200 turns against every other player for every repetition of
+the tournament.
+
+Scores
+------
+
+This gives all the total tournament scores (per player and per repetition)::
+
+    >>> results.scores
+    [[60, 60, 60], [78, 78, 78], [69, 69, 69], [69, 69, 69]]
+
+Normalised scores
+-----------------
+
+This gives the scores, averaged per opponent and turns::
+
+    >>> results.normalised_scores  # doctest: +SKIP
+    [[2.0, 2.0, 2.0], [2.6, 2.6, 2.6], [2.3, 2.3, 2.3], [2.3, 2.3, 2.3]]
+
+We see that Cooperator got on average a score of 2 per turn per opponent.
+Note: Here using string representation to deal with floating point numbers::
+
+    >>> results.normalised_scores[0]
+    [2.0, 2.0, 2.0]
+
+Ranking
+-------
+
+This gives the ranked index of each player::
+
+    >>> results.ranking
+    [1, 2, 3, 0]
+
+The first player has index 1 (:code:`Defector`) and the last has index 0
+(:code:`Cooperator`).
+
+Ranked names
+------------
+
+This gives the player names in ranked order::
+
+    >>> results.ranked_names
+    ['Defector', 'Tit For Tat', 'Grudger', 'Cooperator']
+
+
+Payoffs
+-------
+
+This gives for each player, against each opponent every payoff received for
+each repetition::
+
+    >>> pprint.pprint(results.payoffs)
+    [[[3.0, 3.0, 3.0, 3.0, 3.0, 3.0],
```

Comment:
This will actually need to be changed once @Nikoleta-v3 #635 goes in. Ideally that should go first and then it'll be straightforward to fix this.

Basically: we're currently counting self interactions twice for certain things. This has been fixed in #635.